hexa/workspaces/middlewares.py

```diff
@@ -0,0 +1,38 @@
+import binascii
+from logging import getLogger
+
+from django.core.signing import BadSignature, Signer
+from django.http import HttpRequest
+
+from hexa.workspaces.models import WorkspaceMembership
+
+logger = getLogger(__name__)
+
+
+# FIXME: This needs to be changed as it allows the user to query everything with just a simple workspace token.
```

Comment:
Is this still the case?

Reply:
It clearly is. That's the problem of setting a `request.user` :/